website/src/theme/Blog/Components/Author/Socials/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
what did you change in this file? Looks similar to what we had before 🤔

Reply:
Used swizzle command to extract `Blog/Components/Author`, this nested field was extracted in the process, reverted it.